src/main/java/com/enterprise/fastfoodapplication/service/IFoodService.java

```diff
@@ -17,14 +17,14 @@
      * @param id It is a string (maybe int?) that is compared with foodId
      * @return A Food object containing the details of the food item if found in database, otherwise return NULL
      */
-    Food getFoodItemById(int id);
+    Food getFoodItemById(int id)throws Exception;
```

Comment:
Does it actually throw an exception, though?  If not, this is misleading.  If so, define the specific checked exception it throws.